staff/henry-novoa/cv/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <h1>Curriculum Vitae</h1>
+
+    <section id="personal data">
+      <h2>Personal Data</h2>
+        <p>Name: Henry Novoa</p>
+    <address>
+        <P>Date of birth: 8th of June, 1990</p>
```

Comment:
I think that inside <adress> tags you don't need to use any <p>

Reply:
done!